src/main/java/com/powsybl/openloadflow/network/impl/LfLoads.java

```diff
@@ -101,4 +101,8 @@ private static double getPowerFactor(Load load) {
         return load.getP0() != 0 ? load.getQ0() / load.getP0() : 1;
     }
 
+    public List<Load> getLoads() {
```

Comment:
You cannot do that here, the IIDM model must remained hidden. You have to deal with the information you have at LfBus level `loadTargetP`, and save the state before to restore it after.

Reply:
Done